packages/components/pages/f-account-info/src/components/AccountInfo.vue

```diff
@@ -398,6 +399,27 @@ export default {
             if (isAddressField) {
                 this.hasAddressBeenUpdated = true;
             }
+        },
+
+        /**
+        * Pushes `form` event to the dataLayer with correct data
+        */
+        analyticsTrackFormSubmission (hasAddressBeenUpdated) {
+            this.$gtm.pushEvent({
+                event: 'trackEvent',
+                category: 'account',
+                action: 'save_accountinfo_changes',
+                label: 'submit'
+            });
+
+            if (hasAddressBeenUpdated) {
+                this.$gtm.pushEvent({
+                    event: 'trackEvent',
+                    category: 'my account',
```

Comment:
Should this be `account` like line 410 if not why are they different out of interest and same with label and action? I think the format usual follows underscores  too `my_account`?

Reply:
Unsure all I did was move the methd